Flakey test:  Language randomization causes conflicts

LanguageDataGen creates random languages for tests,  it is random only to 4 different a-z chars e.g. en-us.  
There is some logic to prevent certain known codes to be tried but DeterministicIdentifierAPITest does not use those codes and it seems to be checking the country code instead of language code.   The the language already exists the DeterministicIdentifierAPITest falls back to a unique system time based value causing the test to faile.

Need to fix the check to prevent known "language" codes from being created and include the values that are used in DeterministicIdentifierAPITest.   

Seen in Merge Queue https://github.com/dotCMS/core/actions/runs/9203251769

```
2024-05-23T06:49:39.4772089Z [ERROR] Tests run: 17, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 1.236 s <<< FAILURE! -- in com.dotmarketing.business.DeterministicIdentifierAPITest
2024-05-23T06:49:39.4779086Z [ERROR] com.dotmarketing.business.DeterministicIdentifierAPITest.Test_Language_Deterministic_Id[3: LanguageTestCase{expectedSeed='Language:en:NZ', expectedHash=5382528, langCode='en', countryCode='NZ', country='New Zealand'}] -- Time elapsed: 0.010 s <<< FAILURE!
2024-05-23T06:49:39.4782051Z java.lang.AssertionError: expected:<5382528> but was:<1716446978233>
2024-05-23T06:49:39.4783098Z 	at org.junit.Assert.fail(Assert.java:89)
2024-05-23T06:49:39.4784070Z 	at org.junit.Assert.failNotEquals(Assert.java:835)
2024-05-23T06:49:39.4784807Z 	at org.junit.Assert.assertEquals(Assert.java:647)
2024-05-23T06:49:39.4785666Z 	at org.junit.Assert.assertEquals(Assert.java:633)
2024-05-23T06:49:39.4787359Z 	at com.dotmarketing.business.DeterministicIdentifierAPITest.Test_Language_Deterministic_Id(DeterministicIdentifierAPITest.java:350)
2024-05-23T06:49:39.4789186Z 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
2024-05-23T06:49:39.4790882Z 	at com.tngtech.java.junit.dataprovider.DataProviderFrameworkMethod.invokeExplosively(DataProviderFrameworkMethod.java:119)
2024-05-23T06:49:39.4793589Z 	at com.dotcms.junit.CustomDataProviderRunner.runChild(CustomDataProviderRunner.java:31)
2024-05-23T06:49:39.4796198Z 	at com.dotcms.junit.CustomDataProviderRunner.runChild(CustomDataProviderRunner.java:17)
2024-05-23T06:49:39.4797662Z 	at com.dotcms.junit.MainBaseSuite$DotRunner.run(MainBaseSuite.java:71)
2024-05-23T06:49:39.4826685Z```
[10_Merge Group Test _ JVM IT Tests - JDK 11 MainSuite 2b.txt]
```
(https://github.com/dotCMS/core/files/15418640/10_Merge.Group.Test._.JVM.IT.Tests.-.JDK.11.MainSuite.2b.txt)

--- a/dotcms-integration/src/test/java/com/dotmarketing/portlets/languagesmanager/business/LanguageDataGen.java
+++ b/dotcms-integration/src/test/java/com/dotmarketing/portlets/languagesmanager/business/LanguageDataGen.java
@@ -40,12 +40,12 @@ public class LanguageDataGen extends AbstractDataGen<Language> {
     String country = null;
     
     while(count>0) {
-      country = RandomStringUtils.randomAlphabetic(2);
+      language = RandomStringUtils.randomAlphabetic(2);
       // Skip if the country is one of the ones that we already have
-      if (Set.of("de", "en", "es", "fr", "fi", "nl", "ru", "zh").contains(country)) {
+      if (Set.of("de", "en", "es", "fr", "fi", "nl", "ru", "zh", "ep", "sg").contains(language)) {
         continue;
       }
-      language = RandomStringUtils.randomAlphabetic(2);
+      country = RandomStringUtils.randomAlphabetic(2);
       count = new DotConnect().setSQL("select count(*) as test from language where language_code=? and country_code=?").addParam(language).addParam(country).getInt("test");
 
     }

